### PR TITLE
wheels for windows

### PR DESCRIPTION
**Pull request by pzhokhov**

### CODE REVIEW

File: atari_py/ale_interface/CMakeLists.txt [hunk 1]
```diff
@@ -55,7 +55,9 @@ if(APPLE)
   set(CMAKE_SHARED_LIBRARY_SUFFIX ".so")
 endif()
 
-if(WINDOWS OR MINGW)
+if(WIN32 OR MINGW)
+  # include_directories(${SOURCE_DIR}/zlib)
```
Comment (christopherhesse) [inline atari_py/ale_interface/CMakeLists.txt]: Looks like this commented out line is not necessary

**Review by christopherhesse — APPROVED** [review]
LGTM, only thing missing is a way to keep the pypi upload working

**Comment by JesseFarebro** [comment]
Any idea on how much work is left on this PR or when it will be merged?

**Comment by christopherhesse** [comment]
@JesseFarebro merged now, sorry for the delay!

**Pull request merged**